lib/src/react16_suggestors/react_style_maps_updater.dart

```diff
@@ -0,0 +1,263 @@
+          /// manually checked
+          List<String> listOfVariables = [];
+
+          // Variables that affect the writing of the comment that goes above
+          // the style prop. There are different messages patched based on
+          // these booleans.
+          bool styleMapContainsAVariable = false;
+          bool isAVariable = false;
+          bool isAFunction = false;
+          bool isOther = false;
+          bool isForCustomProps =
+              cascade.toSource().toLowerCase().contains('props');
+
+          if (stylesObject is MapLiteral) {
+            stylesObject.entries.forEach((MapLiteralEntry cssPropertyRow) {
+              final propertyKey = cssPropertyRow.key;
+              String originalCssPropertyKey = propertyKey.toSource();
+              String cleanedCssPropertyKey = cleanString(propertyKey);
+
+              final cssPropertyValue = cssPropertyRow.value;
+              String cleanedCssPropertyValue = cleanString(cssPropertyValue);
+
+              /// Marks the current CSS property as containing a variable.
+              ///
+              /// Used to keep track of what properties in the style map
+              /// contain variables.
+              void flagAsVariable() {
+                if (!listOfVariables.contains(cleanedCssPropertyKey)) {
+                  listOfVariables.add(cleanedCssPropertyKey);
+                }
+
+                if (!styleMapContainsAVariable) {
+                  styleMapContainsAVariable = true;
+                }
+              }
+
+              num end = cssPropertyRow.end;
+
+              // If the codemod does not override the commas originally in
+              // the code, the formatting can be undesirable. As a result,
+              // the codemod manually places a comma after a modified
+              // property. To do this, the script needs to override the
+              // already existing comma (one character after the row end)
+              // without overriding the closing bracket (cascade.end).
+              final nextToken = cssPropertyRow.endToken.next;
+              if (nextToken.type == TokenType.COMMA) {
+                end = nextToken.end;
+              }
+
+              if (cssPropertyValue is ConditionalExpression ||
+                  cssPropertyValue is BinaryExpression) {
+                dynamic ternaryExpressions;
+
+                if (cssPropertyValue is ConditionalExpression) {
+                  ternaryExpressions = [
+                    cssPropertyValue.thenExpression,
+                    cssPropertyValue.elseExpression
+                  ];
+                } else if (cssPropertyValue is BinaryExpression) {
+                  ternaryExpressions = [
+                    cssPropertyValue.leftOperand,
+                    cssPropertyValue.rightOperand
+                  ];
+                }
+
+                ternaryExpressions.forEach((Expression property) {
```

Comment:
Once you add the typing for `ternaryExpressions` on line 93, this can become:

```suggestion
                ternaryExpressions.forEach((property) {
```